### DISCUSSION TITLE
Training through Dreambooth Extension not working

### DISCUSSION THREAD

**Discussion by TomsOsins** [TheLastBen/fast-stable-diffusion#1947] [Q&A]
Hey, i'm trying to train a model through dreambooth extension, but all I get is infinite code like this and steps progress or anything? Any idea on the issue here?

Steps:   0% 0/900 [00:00<?, ?it/s]AUTOTUNE bias_addmm(1x1280, 1x320, 320x1280)
  triton_mm_6 0.0102s 100.0%
  bias_addmm 0.0102s 100.0%
  addmm 0.0133s 76.9%
  triton_mm_8 0.0133s 76.9%
  triton_mm_9 0.0133s 76.9%
  triton_mm_3 0.0133s 76.9%
  triton_mm_1 0.0143s 71.4%
  triton_mm_5 0.0143s 71.4%
  triton_mm_0 0.0164s 62.5%
  triton_mm_2 0.0164s 62.5%
AUTOTUNE bias_addmm(1x1280, 1x1280, 1280x1280)
  bias_addmm 0.0133s 100.0%
  addmm 0.0164s 81.2%
  triton_mm_18 0.0184s 72.2%
  triton_mm_20 0.0215s 61.9%
  triton_mm_21 0.0215s 61.9%
  triton_mm_13 0.0297s 44.8%
  triton_mm_15 0.0297s 44.8%
  triton_mm_17 0.0297s 44.8%
  triton_mm_14 0.0348s 38.2%
  triton_mm_16 0.0348s 38.2%
AUTOTUNE bias_addmm(1x320, 1x1280, 1280x320)
  addmm 0.0133s 100.0%
  triton_mm_30 0.0174s 76.5%
  bias_addmm 0.0174s 76.5%
  triton_mm_29 0.0174s 76.5%
  triton_mm_33 0.0184s 72.2%
  triton_mm_32 0.0195s 68.4%
  triton_mm_25 0.0276s 48.1%
  triton_mm_27 0.0276s 48.1%
  triton_mm_28 0.0276s 48.1%
  triton_mm_26 0.0287s 46.4%
AUTOTUNE mm(4096x320, 320x320)
  triton_mm_37 0.0154s 100.0%
  triton_mm_38 0.0154s 100.0%
  triton_mm_39 0.0154s 100.0%
  triton_mm_40 0.0154s 100.0%
  mm 0.0164s 93.7%
  triton_mm_44 0.0164s 93.7%
  triton_mm_36 0.0174s 88.2%
  triton_mm_43 0.0215s 71.4%
  triton_mm_46 0.0215s 71.4%
  triton_mm_41 0.0225s 68.2%
AUTOTUNE bias_addmm(4096x320, 4096x320, 320x320)
  triton_mm_74 0.0154s 100.0%
  triton_mm_76 0.0154s 100.0%
  triton_mm_73 0.0164s 93.7%
  triton_mm_75 0.0164s 93.7%
  bias_addmm 0.0174s 88.2%
  triton_mm_80 0.0174s 88.2%
  triton_mm_72 0.0184s 83.3%
  triton_mm_79 0.0215s 71.4%
  triton_mm_82 0.0225s 68.2%
  addmm 0.0236s 65.2%
AUTOTUNE mm(77x768, 768x320)
  mm 0.0113s 100.0%
  triton_mm_102 0.0123s 91.7%
  triton_mm_101 0.0123s 91.7%
  triton_mm_105 0.0123s 91.7%
  triton_mm_104 0.0143s 78.6%
  triton_mm_97 0.0195s 57.9%
  triton_mm_98 0.0195s 57.9%
  triton_mm_99 0.0195s 57.9%
  triton_mm_100 0.0195s 57.9%
  triton_mm_96 0.0276s 40.7%
AUTOTUNE bias_addmm(4096x2560, 4096x320, 320x2560)
  bias_addmm 0.0461s 100.0%
  triton_mm_134 0.0522s 88.2%
  triton_mm_135 0.0522s 88.2%
  triton_mm_136 0.0522s 88.2%
  triton_mm_133 0.0532s 86.5%
  triton_mm_132 0.0676s 68.2%
  triton_mm_140 0.0707s 65.2%
  addmm 0.0809s 57.0%
  triton_mm_139 0.0829s 55.6%
  triton_mm_142 0.0932s 49.5%

**Comment by meunumerotim333** [comment]
I have tried before too.. also didn't work. Maybe it's too complex for Colab.

**Reply by TomsOsins** [reply]
I did work on sunday though.

**Comment by codgodtao** [comment]
same as you
